core/jobs.py

```diff
@@ -648,7 +649,9 @@ def run(self, job_id, job_class_str, output):
 
 class GoogleCloudStorageConsistentJsonOutputWriter(
         output_writers.GoogleCloudStorageConsistentOutputWriter):
-    """Google Cloud Storage Consistent Output Writer."""
+    """This is an Output Writer which is used to consistently store MapReduce
```

Comment:
What does "consistently store" mean (as opposed to inconsistently storing it)?

Reply:
Ah ok. Could you please put this explanation + link in the docstring? This would be handy as a reference.

Also, this is the only remaining comment I have for this PR. Everything else LGTM. Thanks!